Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -281,12 +285,48 @@ theorem degrees_esymm [Nontrivial R] (n : ℕ) (hpos : 0 < n) (hn : n ≤ Fintyp
 
 end ElementarySymmetric
 
+section CompleteHomogeneousSymmetric
+
+open Classical Set
```

Comment:
```suggestion
open Set

variable [DecidableEq σ] [DecidableEq τ]
```
should have the same result, and it is more general (don't worry if you don't care about decidability issues).

Reply:
Okay, thanks!